c++/src/kj/exception.c++

```diff
@@ -1263,7 +1282,11 @@ size_t sharedSuffixLength(kj::ArrayPtr<void* const> a, kj::ArrayPtr<void* const>
 
 kj::ArrayPtr<void* const> computeRelativeTrace(
     kj::ArrayPtr<void* const> trace, kj::ArrayPtr<void* const> relativeTo) {
+#ifndef KJ_NO_POSIX
```

Comment:
When an `#else` is present, we prefer the positive case comes first, otherwise the else clause is a double-negative which can be hard to think about. (Actually in this case it's worse, since the flag itself is a negative, so the `#ifndef` is a double-negative and the `#else` is a triple-negative.)

Can you please reverse all the `#ifndef`s that are followed by `#else`?